python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -1589,7 +1592,29 @@ def _fit_astrometry(self, exposure, stars):
         matches : `list` [`lsst.afw.table.ReferenceMatch`]
             Reference/stars matches used in the fit.
         """
+        initialWcs = exposure.wcs
         result = self.astrometry.run(stars, exposure)
+
+        # Record astrometry stats to metadata
+        self.metadata["astrometry_matches_count"] = len(result.matches)
+        if exposure.wcs is not None:
+            if initialWcs is not None:
+                self.metadata['initial_to_final_wcs'] = (
```

Comment:
I suggest using `exposure.getBBox().getCenter()` instead of (0, 0) for the pixel coordinate.